tests/postgres_tests/test_hstore.py

```diff
@@ -37,6 +39,17 @@ def test_value_null(self):
         reloaded = HStoreModel.objects.get()
         self.assertEqual(reloaded.field, value)
 
+    def test_key_val_cast_to_string(self):
+        value = {'a': 1, 'b': 'B', 2: 'c', 'd': 'ê'}
+        expected_value = {'a': '1', 'b': 'B', '2': 'c', 'd': force_text('ê')}
```

Comment:
We should check the behavior with bytestrings too. I'm not sure how PostgreSQL stores them and if the change to force them to unicode will have any practical consequences.